packages/core/docs/advanced/logging.md

```diff
@@ -44,44 +47,56 @@ If not explicitly changed, logging level depends on current environment variable
 
 ## Writing custom loggers
 
-By default we're printing all the events happening in the app in the console  but you can easily write a new logger and use a third-party library (like `consola`) or pass the logs to external service like `Sentry.io`
+By default, we're printing all the events happening in the app in the console but you can easily write a new logger and use a third-party library (like [consola](https://github.com/nuxt-contrib/consola)) or pass the logs to the external service like [Sentry](https://sentry.io/welcome/)
 
-To override the default logger simply pass a function to the `logger.customLogger` property of `@vue-storefront/nuxt` module. This function returns the logger object and as an argument, you have access to the `verbosity` level.
-
-If you are using our nuxt module, setting up the logger is much simpler. All you have to do is just provide the configuration to the module:
+To override the default logger, pass a function to the `logger.customLogger` property of `@vue-storefront/nuxt` module. This function returns the logger object and as an argument, you have access to the `verbosity` level.
```

Comment:
@filrak we definitely need a code example here

Reply:
this refers to line 57, perhaps this description is just obsolete (at least part of this that describes the example)

Reply:
I removed an unnecessary explanation here 👍🏻